index.js

```diff
@@ -332,7 +344,9 @@ class Plugin extends BtpPlugin {
       balance: this.baseToXrp(this._bestClaim.amount),
       channel: this._clientChannel,
       signature: this._bestClaim.signature.toUpperCase(),
-      publicKey: this._paychan.publicKey
+      publicKey: this._paychan.publicKey,
+    }, {
+      maxFeeXrp
```

Comment:
Isn't this property called `maxFee`? (https://developers.ripple.com/rippleapi-reference.html#transaction-instructions)